exception_al/user/urls.py

```diff
@@ -3,5 +3,7 @@
 
 urlpatterns = [
     path('users/', views.CustomUserList.as_view()),
-    path('users/<int:pk>/', views.CustomUserDetail.as_view()),
+    path('user/register', views.CustomUserRegister.as_view()),
+    path('user/<int:pk>/', views.CustomUserDetail.as_view()),
+    path('api-token-auth/', views.CustomAuthToken.as_view(), name='api_token_auth')
```

Comment:
This line of code can be moved to the project's url file!

Reply:
Thank you for checking  @suanTech. I've removed that duplicate url from the users/urls.py file.

Reply:
![image](https://github.com/SheCodesAus/2024_exception_al_back_end/assets/146698245/2e346b96-2425-42d9-9e6c-a95f872c6147)